mw-webapp/src/component/header/Header.tsx

```diff
@@ -39,12 +39,45 @@ export const Header = () => {
           onClick={user ? logOut : logIn}
           value={user ? BUTTON_LOG_OUT_VALUE : BUTTON_LOG_IN_VALUE}
         />
-        {/* //TODO: we need delete this link when after sidebar with all links to al pages will be added */}
-        <Link
-          path={pages.way.path("uuid")}
-          value={LINK_TEXT}
+        <Sidebar
+          trigger={
+            <Button value={"Navigation"} />
+          }
```

Comment:
Can we use as trigger any element? What do you think about using just text instead of button? Or it's a bad idea?

Reply:
Or we can use burgermenu for open sidebar? Dk how is better

Reply:
Agreed, its good enough for now. Let's focus on code quality and core features.

Reply:
Okey, let's skeep it now